entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -26,9 +26,9 @@ SQLite doesn't natively support the following data types. EF Core can read and w
 * TimeSpan
 * UInt64
 
-Instead of `DateTimeOffset`, we recommend using DateTime values. When handling multiple time zones, we recommend converting the values to UTC before saving and then converting back to the appropriate time zone.
+Instead of `DateTimeOffset`, we recommend using `DateTime` values. When handling multiple time zones, we recommend converting the values to UTC before saving and then converting back to the appropriate time zone.
 
-The `Decimal` type provides a high level of precision. If you don't need that level of precision, however, we recommend using double instead. You can use a [value converter](xref:core/modeling/value-conversions) to continue using decimal in your classes.
+The `Decimal` type provides a high level of precision. If you don't need that level of precision, however, we recommend using `Double` instead. You can use a [value converter](xref:core/modeling/value-conversions) to continue using `Decimal` in your classes.
```

Comment:
nit: I think the built-in lower-case names are a bit better, that's how they're usually referred to in .NET (but obviously OK either way).

```suggestion
The `decimal` type provides a high level of precision. If you don't need that level of precision, however, we recommend using `double` instead. You can use a [value converter](xref:core/modeling/value-conversions) to continue using `decimal` in your classes.
```